test/e2e/lifecycle/cluster_upgrade.go

```diff
@@ -449,3 +462,55 @@ func getUpgradeContext(c discovery.DiscoveryInterface, upgradeTarget string) (*u
 
 	return upgCtx, nil
 }
+
+//  remove 1.10 storage protection finalizer from PV and PVC objects
+func removeStorageProtectionFinalizers(c clientset.Interface) error {
+	pvs, err := c.CoreV1().PersistentVolumes().List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, pv := range pvs.Items {
+		err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+			pv, err := c.CoreV1().PersistentVolumes().Get(pv.Name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			finalizers := sets.NewString(pv.Finalizers...)
+			if !finalizers.Has("kubernetes.io/pv-protection") {
+				return nil
+			}
+			finalizers.Delete("kubernetes.io/pv-protection")
+			pv.Finalizers = finalizers.List()
+			_, err = c.CoreV1().PersistentVolumes().Update(pv)
+			return err
+		})
+		if err != nil && !apierrors.IsNotFound(err) {
+			return err
+		}
+	}
+
+	pvcs, err := c.CoreV1().PersistentVolumeClaims("").List(metav1.ListOptions{})
```

Comment:
no pvc namespace?

Reply:
correct, list across all namespaces

Reply:
`metav1.NamespaceAll` would be more clear